src/main/java/seedu/dengue/logic/parser/ParserUtil.java

```diff
@@ -42,12 +45,10 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
      * @throws ParseException if the given {@code name} is invalid.
      */
     public static Name parseName(String name) throws ParseException {
-        requireNonNull(name);
-        String trimmedName = name.trim();
-        if (!Name.isValidName(trimmedName)) {
+        if (!PredicateUtil.isNameValid(name)) {
```

Comment:
Minor nitpick: think the method might be better named `isValidName` rather `isNameValid`! But `isNameValid` gets the message across too

Same with the other methods like isAgeValid too :)